### PR TITLE
WM-2658: Workflows app status panel POC

### PR DESCRIPTION
**Pull request by cjllanwarne**
## Summary of changes:
- Adds a Workflow Apps Status panel in analogy to the Data Table Status panel.
- Behind a feature flag to emphasise its use as a developer / maintainer tool, not necessarily user facing

### What
Workflow status app panel - 

image

### Why
To help developer debugging and validation during upgrades

### Testing strategy
Tested in a working workspace.
Tested in a workspace with a bad CBAS instance, which was correctly reported.

### CODE REVIEW

File: src/libs/feature-previews-config.ts [hunk 1]
```diff
@@ -118,6 +119,14 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
       'Feedback on deprecating Firecloud UI'
     )}`,
   },
+  {
+    id: WORKFLOWS_APP_STATUS,
+    title: 'Workflows App Status Modal',
+    description: 'Enabling this feature will allow you to view the status of the workflows app.',
+    feedbackUrl: `mailto:dsp-workflow-management@broadinstitute.org?subject=${encodeURIComponent(
+      'Feedback on deprecating Firecloud UI'
```
Comment (aednichols) [inline src/libs/feature-previews-config.ts]: Copy-pasta

File: src/libs/feature-previews-config.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ export const ENABLE_WORKFLOW_RESOURCE_MONITORING = 'enableWorkflowResourceMonito
 export const ENABLE_AZURE_PFB_IMPORT = 'enableAzurePfbImport';
 export const ENABLE_AZURE_TDR_IMPORT = 'enableAzureTdrImport';
 export const FIRECLOUD_UI_MIGRATION = 'firecloudUiMigration';
+export const WORKFLOWS_APP_STATUS = 'workflowsAppStatus';
```
Comment (aednichols) [inline src/libs/feature-previews-config.ts]: IMO we can be a bit too shy about putting features out there that are clearly helpful but maybe not perfect. I would personally not feature flag this.

**Review by aednichols — APPROVED** [review]
Love it!

Could bike-shed some more features in a future PR (e.g. chart version)

**Comment by cjllanwarne** [comment]
>Could bike-shed some more features in a future PR (e.g. chart version)

Yes, chart version would be awesome, as would the app version. The main reason that's not part of this is that it's not trivially available via our APIs yet (or gets blocked by the relay). Small fixes but not in this POC.

File: src/libs/feature-previews-config.ts (hunk 2 above)
Comment (cjllanwarne) [reply src/libs/feature-previews-config.ts]: Seems like Sara agrees, so you may well get your wish!

File: src/workflows-app/components/WorkflowsAppNavPanel.ts [hunk 3]
```diff
@@ -242,6 +244,24 @@ export const WorkflowsAppNavPanel = ({
             ),
           ]
         ),
+        h(
+          Clickable,
+          {
+            'aria-label': 'troubleshooting-button',
+            style: {
+              ...styles.subHeaders(false),
+              // color: colors.accent(1.1),
```
Comment (salonishah11) [inline src/workflows-app/components/WorkflowsAppNavPanel.ts]: Should this comment be removed?

**Review by salonishah11 — APPROVED** [review]

**Pull request merged**